storage/tests/tests/test_compact.py

```diff
@@ -6,27 +6,31 @@
 
 
 def test_compact():
-    sc, sp = common.init(unlock=True)
-    for s in (sc, sp):
-        s.set(0xBEEF, b"hello")
-        s.set(0xBEEF, b"asdasdasdasd")
-        s.set(0xBEEF, b"fsdasdasdasdasdsadasdsadasdasd")
-        s.set(0x0101, b"a" * (consts.NORCOW_SECTOR_SIZE - 600))
-        s.set(0x03FE, b"world!")
-        s.set(0x04FE, b"world!xfffffffffffffffffffffffffffff")
-        s.set(0x05FE, b"world!affffffffffffffffffffffffffffff")
-        s.set(0x0101, b"s")
-        s.set(0x06FE, b"world!aaaaaaaaaaaaaaaaaaaaaaaaab")
-        s.set(0x07FE, b"worxxxxxxxxxxxxxxxxxx")
-        s.set(0x09EE, b"worxxxxxxxxxxxxxxxxxx")
-    assert common.memory_equals(sc, sp)
+    for byte_access in (
+        True,
+        False,
```

Comment:
Is there any reason not to use `@pytest.mark.parametrize` like we do in device tests?

Reply:
refactored https://github.com/trezor/trezor-firmware/commit/7ad1c6ae1e81f4975045b862c627881ec1d70de3